.github/workflows/e2e-tests.yaml

```diff
@@ -59,6 +60,7 @@ jobs:
           kubectl get pods -n kube-system
           echo "kubectl config current-context:" $(kubectl config current-context)
           echo "KUBECONFIG env var:" ${KUBECONFIG}
+          kubectl get storageclass
```

Comment:
@ishankhare07 I think we should remove this debug output here

Reply:
addressed the comments in the latest commit @FabianKramm